### ISSUE TITLE
ubuntu 18.04 beta libva1

### ISSUE THREAD

**Issue by ubuntushop** [stockmind/gpd-pocket-ubuntu-respin#114]
Did a test with ubuntu 18.04 beta iso but fails with:
./isorespin.sh: Package 'libva1' not found
and this package is not installable in ubuntu 18.04 (only with downloading deb and dpkg-i)
is this package needed?

**Comment by petrmatula190** [comment]
You do not need create isorespin ISO. Use regular daily build ISO and add manually boot parameters in grub.After install, download this rep and install kernel update-kernel.sh and update.sh. After reboot, all will be in same state like install from modified ISO.
I am testing it and only sound have some problem, check my bug from start of this week.

**Comment by YetiElvis** [comment]
I am also trying to install ubuntu 18.04 beta. But when using the regular daily build ISO i end up with a blank screen after choosing either to live-boot or to install off the usb directly.
Any idea what i'm doing wrong?

**Comment by petrmatula190** [comment]
@YetiElvis Pres ESC when start boot from untouched ISO and search key for edit boot cmd line (if I boot in virtualbox, key is F6. If you have EFI boot directly on Pocket HW, it can be E.
Add this oprions on cmdline and boot

`i915.fastboot=1 i915.semaphores=1 fbcon=rotate:1`

**Comment by stockmind** [comment]
Build 18.04 iso passing `bionicbeaver` parameter to respin script like:

```
./build.sh <iso filename> gnome bionicbeaver
```
(don't forget the `gnome` required parameter for default Ubuntu 17.10+)

So for 18.04 beta 2 iso it should be something like:

```
./build.sh ubuntu-18.04-beta2-desktop-amd64.iso gnome bionicbeaver
```

Or if you want to package a Unity version this should work:

```
./build.sh ubuntu-18.04-beta2-desktop-amd64.iso unity bionicbeaver
```

**Comment by ubuntushop** [comment]
Installation is working ok, but fails on grub efi installation. Maybe because it is still beta.
Installer does not see that there is efi boot.
gd

**Comment by stockmind** [comment]
Are you installing Ubuntu on live environment with internet connection enabled and working?

Are you installing with “Check and install updates” enabled during install?

I had sometimes problems like that installing from Live environment without a working internet connection, It seems that sometimes the installer needs internet to download some packages to let GRUB install correctly.

**Comment by ubuntushop** [comment]
Yes, internet is working,updates during install checked and also when unchecked.
Log files from installer are minimal, i am trying to install with debug-ubiquity to get better logs.

**Comment by ubuntushop** [comment]
Yes, internet is working,updates during install checked and also when unchecked.
Log files from installer are minimal, i am trying to install with debug-ubiquity to get better logs.
log file:
http://95.211.190.99/debug.txt

**Comment by ubuntushop** [comment]
syslog has also info:
http://95.211.190.99/syslog

**Comment by ubuntushop** [comment]
I can confirm that the same iso is working ok on a uefi enabled laptop
(ubuntu-18.04-beta2-desktop-amd64.iso)

**Comment by stockmind** [comment]
@ubuntushop i had time to look better into logs, great job!
The only odd thing is that GRUB output error show that it doesn't recognize the EFI system. Are you sure you booted as EFI and not legacy?

```
EFI variables are not supported on this system.
EFI variables are not supported on this system.
grub-install: error: efibootmgr failed to register the boot entry: No such file or directory.
```

I should have time in the next days to test a clean install.

**Comment by ubuntushop** [comment]
It was on the GPD, i do not think you can disable Uefi.
But i cannot reproduce: i have respinned the official ubuntu 18.04 but the installer crashes very early.
I will look for the error

**Comment by ubuntushop** [comment]
it's a ubiquity xwindows error:
http://95.211.190.99/debug
http://95.211.190.99/syslog

**Comment by Morantz** [comment]
@ubuntushop I had the ame isues with installer crashing very early.  I have now been able to respin ubuntu and xubuntu 18.04 ISOs.  The fix?  A reboot....  I didn't see anything different in the respin logs, but ISOs created are making it to the EFI install error again...

**Comment by hryamzik** [comment]
@ubuntushop did you manage to solve this?

I face the same error but my disk setup may be a bit unusual, I've shrinked windows volume, added `/boot` `ext4` `512MB` to a free space as a primary partition and added another primary partition, crypt, `ext4` at `/`.

I'm also not sure if turning off fast startup and/or secure boot in windows could make difference.

**Comment by hryamzik** [comment]
Tried with gnome version – no lock as well, installer crashes. However original Ubuntu ISO works fine with the same options, looks like it's the best way to get system installed.

**Comment by elcuervo** [comment]
I ended up here as well the solution it's just adding `bionicbeaver` somewhere in the args:

https://github.com/stockmind/gpd-pocket-ubuntu-respin/blob/master/build.sh#L21-L24
